Readme.md

```diff
@@ -32,7 +32,39 @@ arrays, and deserialization from arrays to Rust structs.
 [datafusion]: https://github.com/apache/arrow-datafusion/
 
 ## Example
+Given this Rust structure
+```rust
+#[derive(Serialize, Deserialize)]
+struct Record {
+    a: f32,
+    b: i32,
+}
+
+let records = vec![
+    Record { a: 1.0, b: 1 },
+    Record { a: 2.0, b: 2 },
+    Record { a: 3.0, b: 3 },
+];
+```
 
+### Serialize to `arrow` `RecordBatch`
```

Comment:
I would prefer the arrow implementations as headings with the conversion and the parquet io under a single heading. More or less exactly the same as in the `lib.rs`.